lib/tasks/migrate.rake

```diff
@@ -220,11 +220,19 @@ namespace :host_reports do
   This task refreshes all host statuses and global statuses.
   END_DESC
   task :refresh => :environment do
+    # ensure the class is loaded before refresh starts to avoid hash modifycation during loop
+    HostStatus::HostReportStatus
```

Comment:
I think this is causing "can't add a new key into hash during iteration"

Reply:
Tiny nitpick: modification?